src/pages/transactions/[id].vue

```diff
@@ -44,12 +49,34 @@ const {
   transactionDetails,
   transactionTypeData,
 } = storeToRefs(transactionDetailsStore)
-const { fetchTransactionDetails } = transactionDetailsStore
+const {
+  fetchTransactionDetails,
+} = transactionDetailsStore
 const route = useRoute()
-
+const { subscribedTransactionId } = storeToRefs(useWebSocket())
 const { isLoading } = useLoading()
 
 await fetchTransactionDetails(route.params.id)
+
+const isSyncing = computed(() => isLoading.value || !transactionTypeData.value)
+
+if (process.client && !transactionTypeData.value) {
+  subscribedTransactionId.value = route.params.id
+
+  onBeforeRouteUpdate(
+    (_to, _from, next) => {
+      subscribedTransactionId.value = null
+      next()
+    },
+  )
+
+  onBeforeRouteLeave(
+    (_to, _from, next) => {
```

Comment:
These params are not needed I guess

Reply:
well... how else can I get to next without putting two placeholders first? :)

Reply:
Ok, I see now.

I quess this is not gonna work?

```suggestion
    ({next}) => {
```

But the `_` prefix could be removed, right? Or why is there underscore?